infra/terraform/test-org/org/gsuite.tf

```diff
@@ -99,8 +103,61 @@ resource "google_billing_account_iam_member" "ci_gsuite_sa_billing" {
   member             = "serviceAccount:${google_service_account.ci_gsuite_sa.email}"
 }
 
-# Grant G-Suite project rights to cft_ci_group
-# Required to be able to create keys for the gsuite sa.
+// Generate a json key and put it into the secrets bucket.
+
+resource "google_service_account_key" "ci_gsuite_sa" {
+  service_account_id = google_service_account.ci_gsuite_sa.id
+}
+
+provider "google" {
+  alias       = "sa"
+  credentials = base64decode(google_service_account_key.ci_gsuite_sa.private_key)
+}
+
+resource "google_storage_bucket" "ci_gsuite_sa" {
```

Comment:
Why does this bucket need to be created with a different provider? The normal/default one should work fine.